testsuite/classlibrary/TestNodeProxy_Server.sc

```diff
@@ -54,6 +54,19 @@ TestNodeProxy_Server : UnitTest {
 		this.assertEquals(build, false, "SynthDef should not be rebuilt when calling send");
 	}
 
+	test_synthDef_isReleased_afterFree {
+		var numBefore, numAfter;
+		numBefore = server.statusWatcher.numSynthDefs;
```

Comment:
I realise this may be being ridiculously pedantic, but to be super safe, would it make sense to include a status message and sync before checking the number of defs? Or is it impossible that some other test might affect that? I'm a bit hazy on the current state of how the test suite runs, but I think that is at least slightly possible. (Ideally we could query the server to see if it still knows the def, but in the absence of another use case that seems like bloat, and anyway out of scope for this PR.)

Reply:
Probably! I'm not sure what you mean by "include".

we probably should have a `ServerUnitTest` class with its own `sync` method in order to maintain such guarantees?

Reply:
I just meant add an explicit status call and sync before the node proxy stuff so we know we’re up to date. (Well ‘know’)

Reply:
I just meant add an explicit status call and sync before the node proxy stuff so we know we’re up to date. (Well ‘know’)

Reply:
I see, yes that is better. Thanks!